pubspec.yaml

```diff
@@ -5,7 +5,7 @@ description: APIs for parsing and manipulating HTML content outside the browser.
 homepage: https://github.com/dart-lang/html
 
 environment:
-  sdk: '>=2.3.0 <3.0.0'
+  sdk: '>=2.8.0 <3.0.0'
```

Comment:
should this be 2.8.1? or should the travis jobs be updated to 2.8.0?

Reply:
It won't end up mattering much, I just bumped to to an arbitrary version so analyzer wouldn't complain about the use of the extension. This will end up bumped to 2.11.0 for the null safety migration and I don't see a need to publish in between.